crates/ruff_linter/src/checkers/ast/analyze/expression.rs

```diff
@@ -1404,6 +1404,9 @@ pub(crate) fn expression(expr: &Expr, checker: &mut Checker) {
             if checker.enabled(Rule::IfExpInsteadOfOrOperator) {
                 refurb::rules::if_exp_instead_of_or_operator(checker, if_exp);
             }
+            if checker.enabled(Rule::RedundantTernary) {
```

Comment:
I quickly checked other "ternary" specific rules and all of them use the term `if-expr` instead. We also use the term `Useless` when referring to *useless* code.

How about: `useless-if-expr`?

Reply:
`uesless-if-expr` sounds good to me. I agree that that's more approachable language than `ternary`

Reply:
Sounds good, but if we also cover if blocks maybe we will need a different name. For example: `useless-if-logic`

Reply:
hmm, good point. Maybe `useless-if-else`?

Reply:
I like`useless-if-else`

Reply:
I like `useless-if-else`. It provides room for expanding the rule in the future.